src/controllers/playback/video/index.js

```diff
@@ -1227,50 +1233,22 @@ export default function (view) {
                     e.stopPropagation();
                 }
                 break;
-            case 'k':
-                playbackManager.playPause(currentPlayer);
-                showOsd(btnPlayPause);
-                break;
             case 'ArrowUp':
-            case 'Up':
                 playbackManager.volumeUp(currentPlayer);
                 break;
             case 'ArrowDown':
-            case 'Down':
                 playbackManager.volumeDown(currentPlayer);
                 break;
-            case 'l':
```

Comment:
IMO, it is better to keep one command handler - less code to maintain. Also, there is no harm in preventing default action for arrow keys.